File: src/main/java/com/dotronglong/faker/config/FakerApplicationProperties.java
package com.dotronglong.faker.config;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Getter
@Setter
@Configuration
@ConfigurationProperties(prefix = "faker")
public class FakerApplicationProperties {
    private String source;
    private String route;
}

File: src/main/kotlin/com/dotronglong/faker/config/FakerApplicationProperties.kt
package com.dotronglong.faker.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "faker")
data class FakerApplicationProperties(
        var source: String = "",
        var watch: Boolean = false,
        var version: String = "dev"
)